### PR TITLE
fix: updtate twitter logo to x

### LINKED ISSUE #8994: Twitter logo needs to be updated to X
### Description
in the footer of [docs.meshery.io](https://docs.meshery.io/) twitter's old logo is needs to be updated to X

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by akagami-harsh**
**Notes for Reviewers**

This PR fixes #
- update twitter logo to x 
- resolves #8994 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5d428b526ca4499c764616fcb5320c04bb3260b5 at: https://653238ee1529747a56856191--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fd82a857afe36b51816e5914bf7809c83d6f7c02 at: https://653243723cc9eb029b3ab134--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fd82a857afe36b51816e5914bf7809c83d6f7c02 at: https://6532437c2d9aba7fd921e41e--meshery-docs-preview.netlify.app

File: ui/package-lock.json
Comment (leecalcote) [inline ui/package-lock.json:1]: Will you undo / remove changes to the package-lock.json?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks for this PR, @akagami-harsh 👍

File: ui/package-lock.json
Comment (akagami-harsh) [reply ui/package-lock.json:1]: Done!

**Comment by l5io** [comment]
🚀 Preview for commit 0cc9fc5a81a7b0ac37ac44d3d49620c17c8d623a at: https://653755397b046325e68d417b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit faa3070992a12fa91192d184aa3b35765796503e at: https://65375ec813f95e2d6590ed6c--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
on [homepage](https://docs.meshery.io): 
Screenshot 2023-10-24 114856

**Comment by l5io** [comment]
🚀 Preview for commit aef7531a532a4859491feef074ea37bd0929c694 at: https://653764424b582932576e3795--meshery-docs-preview.netlify.app

**Review by vishalvivekm — APPROVED** [review]
LGTM.
Thanks @akagami-harsh

**Review by Yashsharma1911 — CHANGES_REQUESTED** [review]
nice work @akagami-harsh, it seems X logo is small compared to other logos, can you make sure it is similar in height and width to other logos

Screenshot 2023-10-24 at 2 50 41 PM

**Comment by l5io** [comment]
🚀 Preview for commit e10c63aa4ac0177a5cc83d710724817fb0e06506 at: https://653799e89d171b12fff2259a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e4f96b7079f1608b9b337ce535e083dd6c1aa876 at: https://6537a3ed3bd3db19df34c649--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bcf6ff00a13e9f24b17ade42cdf50b9e2f06aafb at: https://6537aca98fdc6b216a9ef135--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 42d554804c22a2b2c93ff7021d3cf25af830a547 at: https://6537eb64313ef6142491c1a0--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow at 7:30 pm IST / 9:00 am CT,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by l5io** [comment]
🚀 Preview for commit 59d2eb70a4e31d1a863855ecd72dd9e064ec604b at: https://65397e079c5ddb2ce7b347af--meshery-docs-preview.netlify.app

**Review by Yashsharma1911 — APPROVED** [review]
Thanks @akagami-harsh LGTM

**Pull request merged**